README.md

```diff
@@ -145,6 +145,38 @@ You got the idea.
 	shoot update                          # Update browser list (WARNING: will override active browsers)
 	shoot version, --version, -v          # Shoot version
 
+### Using ngrok
+
+In order to access your local development environment on BrowserStack you need to forward it somehow to the external work (a.k.a. the internet). BrowserStack has it's own forwarded, but ngrok is better. If you wanna use it:
+
+1. Install it from [https://ngrok.com/download](https://ngrok.com/download)
+
+2. Enable subdomains by registering.
+
+3. Use the `Shoot::Ngrok` class in your test, like this:
+
+``` ruby
+  def my_test
+    my_server = Shoot::Ngrok(12345)
```

Comment:
Would it be possible to run multiple Ngrok servers? (We sometimes need another one for PGW on a different port)